### ISSUE TITLE
$sampleRate data type

### ISSUE THREAD

**Issue by epifrin** [domnikl/statsd-php#62]
Hi there!

Is it correct that you annotate variable $sampleRate like integer?

```php
/**
     * increments the key by 1
     *
     * @param string $key
     * @param int $sampleRate
     * @param array $tags
     */
    public function increment($key, $sampleRate = 1, $tags = [])
    {
        $this->count($key, 1, $sampleRate, $tags);
    }
```
As I know sample rate can be in range 0-1
[https://github.com/etsy/statsd/blob/master/docs/metric_types.md#sampling](https://github.com/etsy/statsd/blob/master/docs/metric_types.md#sampling)

Thanks

**Comment by domnikl** [comment]
fixed in 599393391ed56ea99227941befc05bfb600b554e

**Comment by epifrin** [comment]
Thank you!
